be/src/common/config.h

```diff
@@ -519,7 +519,9 @@ CONF_Int32(query_max_memory_limit_percent, "90");
 CONF_Double(query_pool_spill_mem_limit_threshold, "1.0");
 CONF_Int64(load_process_max_memory_limit_bytes, "107374182400"); // 100GB
 CONF_Int32(load_process_max_memory_limit_percent, "30");         // 30%
-CONF_mBool(enable_new_load_on_memory_limit_exceeded, "true");
+// It's hard limit for loading, when this limit is hit, new loading task will be rejected.
+CONF_mInt32(load_process_max_memory_hard_limit_percent, "60"); // 60%
+CONF_mBool(enable_new_load_on_memory_limit_exceeded, "false");
```

Comment:
so what's the difference between these two configurations. What's the behavior.

Reply:
`load_process_max_memory_limit_percent` is a soft limit, when this limit is hit, memtable in delta writer will be flush to reduce memory cost. Load memory beyond this limit is allowed.
`load_process_max_memory_hard_limit_percent` is a hard limit, when this limit is hit, new loading task will be rejected.

Reply:
Add comments